js_modules/dagster-ui/packages/ui-components/src/components/PageHeader.tsx

```diff
@@ -25,8 +25,9 @@ export const PageHeader = (props: Props) => {
     >
       {title && (
         <Box
+          padding={{vertical: 8}}
```

Comment:
This is a global change but I verified it on a bunch of pages - the padding only takes effect when the height of the content in this box exceeds 36px, which is only the case when it is wrapping to two lines.

Reply:
Is it worth making this its own PR? It might be annoying to have to do a revert if this breaks something unexpectedly.

Reply:
Ahh I think you're right, I'll break this out into a separate PR